app/src/sharedTest/java/org/oppia/android/app/databinding/TextViewBindingAdaptersTest.kt

```diff
@@ -140,37 +140,91 @@ class TextViewBindingAdaptersTest {
+    fakeOppiaClock.setCurrentTimeMs(TIMESTAMP)
+    val getCurrentTimeMs = fakeOppiaClock.getCurrentTimeMs()
+    activityRule.scenario.onActivity {
+      val textView: TextView = it.findViewById(R.id.test_text_view)
       setProfileLastVisitedText(
         textView,
         /* setText= */ getCurrentTimeMs - 60000
       )
       assertThat(textView.text.toString()).isEqualTo("Last used a minute ago")
+    }
+  }
 
+  @Test
+  fun testTextViewBindingAdapters_ltrIsEnabled_port_profileLastVisitedTextIsCorrectCase3() {
+    fakeOppiaClock.setFakeTimeMode(FakeOppiaClock.FakeTimeMode.MODE_FIXED_FAKE_TIME)
+    fakeOppiaClock.setCurrentTimeMs(TIMESTAMP)
+    val getCurrentTimeMs = fakeOppiaClock.getCurrentTimeMs()
+    activityRule.scenario.onActivity {
+      val textView: TextView = it.findViewById(R.id.test_text_view)
       setProfileLastVisitedText(
         textView,
         /* setText= */ getCurrentTimeMs - 120000
       )
       assertThat(textView.text.toString()).isEqualTo("Last used 2 minutes ago")
+    }
+  }
 
+  @Test
+  fun testTextViewBindingAdapters_ltrIsEnabled_port_profileLastVisitedTextIsCorrectCase4() {
+    fakeOppiaClock.setFakeTimeMode(FakeOppiaClock.FakeTimeMode.MODE_FIXED_FAKE_TIME)
+    fakeOppiaClock.setCurrentTimeMs(TIMESTAMP)
+    val getCurrentTimeMs = fakeOppiaClock.getCurrentTimeMs()
+    activityRule.scenario.onActivity {
+      val textView: TextView = it.findViewById(R.id.test_text_view)
       setProfileLastVisitedText(
         textView,
         /* setText= */ getCurrentTimeMs - 3600000
       )
       assertThat(textView.text.toString()).isEqualTo("Last used an hour ago")
+    }
+  }
 
+  @Test
+  fun testTextViewBindingAdapters_ltrIsEnabled_port_profileLastVisitedTextIsCorrectCase5() {
+    fakeOppiaClock.setFakeTimeMode(FakeOppiaClock.FakeTimeMode.MODE_FIXED_FAKE_TIME)
+    fakeOppiaClock.setCurrentTimeMs(TIMESTAMP)
+    val getCurrentTimeMs = fakeOppiaClock.getCurrentTimeMs()
+    activityRule.scenario.onActivity {
+      val textView: TextView = it.findViewById(R.id.test_text_view)
       setProfileLastVisitedText(
         textView,
         /* setText= */ getCurrentTimeMs - 7200000
       )
       assertThat(textView.text.toString()).isEqualTo("Last used 2 hours ago")
+    }
+  }
 
+  @Test
+  fun testTextViewBindingAdapters_ltrIsEnabled_port_profileLastVisitedTextIsCorrectCase6() {
+    fakeOppiaClock.setFakeTimeMode(FakeOppiaClock.FakeTimeMode.MODE_FIXED_FAKE_TIME)
+    fakeOppiaClock.setCurrentTimeMs(TIMESTAMP)
+    val getCurrentTimeMs = fakeOppiaClock.getCurrentTimeMs()
+    activityRule.scenario.onActivity {
+      val textView: TextView = it.findViewById(R.id.test_text_view)
       setProfileLastVisitedText(
         textView,
         /* setText= */ getCurrentTimeMs - 86400000
       )
       assertThat(textView.text.toString()).isEqualTo("Last used yesterday")
+    }
+  }
 
+  @Test
+  fun testTextViewBindingAdapters_ltrIsEnabled_port_profileLastVisitedTextIsCorrectCase7() {
```

Comment:
Rather than case1, case2, etc. you should name this based on the conditions and outcome, per https://github.com/oppia/oppia-android/wiki/Oppia-Android-Testing#naming-convention.

For this specific method, it could be called:
``testSetProfileLastVisitedText_forTimeTwoDaysAgo_setsLastUsedTwoDaysAgoText``

Ditto elsewhere.

Reply:
Done.